components/gfx/font.rs

```diff
@@ -253,7 +254,7 @@ impl Font {
             .borrow_mut()
             .entry(lookup_key)
             .or_insert_with(|| {
-                let start_time = time::precise_time_ns();
```

Comment:
you can remove `time = { workspace = true }` now from `components/gfx/Cargo.toml`